### PR TITLE
Intro to monitoring

### PR DESCRIPTION
**Pull request by claireriley**
This is my first monitoring-centric page draft ready for review (@ArjunSingh-PM since the UI is being updated shortly, it made the most sense for me to start here). 

This addition walks the user through the monitor dashboard and monitoring usage dashboard (not quite sure where to use monitor versus monitoring, let me know if anything sounds awkward) — two gaps that will be addressed once this is merged. I also rewrote the vast majority and reworked it to fit our style guide. 

Let me know what y'all think and happy weekend!

### CODE REVIEW

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 1]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
+All plans, paid and free, are able to use Postman's monitoring service. To find an account type's usage limits, refer to your [usage overview](https://go.pstmn.io/postman-account-limits) or our [pricing](https://www.getpostman.com/pricing).
 
-Each Postman user gets 1,000 monitoring calls for free per month. Paid teams have higher limits: Postman Pro teams have 10,000 monthly monitoring calls included, and Enterprise teams have 100,000 free monthly requests. It is very simple to set up a monitor. Learn more about [monitoring pricing](/docs/postman/monitors/pricing-monitors/) and [getting started with monitors](/docs/postman/monitors/setting-up-monitor/).
+When a free Postman account reaches their usage limit for monitoring, they'll be automatically capped for the remainder of your monthly cycle.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/postman/monitors/intro-monitors.md]: When users on a free Postman account reach the monitoring usage limit of 1,000 monitoring calls per month, they will be required to upgrade to the pro Postman account to continue to use monitoring for the remainder of their monthly cycle.

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 2]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
 
-**Note:** Setting up monitors by minutes is not available for free users. If you are a free user, you can schedule your monitor runs on an hourly, daily or a weekly basis.
+Paid Postman accounts have the option to cap their monthly monitoring usage, but will automatically have pay-as-you-go pricing at a rate of $0.75/1,000 calls beyond their monthly limit. Paid teams can also opt for block pricing, which is $20 per pack of 50,000 calls, per month.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/postman/monitors/intro-monitors.md]: Paid Postman accounts have the option to cap their monthly monitoring usage by deselecting the "Allow monitoring overages" from the Billing->Overages section. However, this option is selected by default and the user will be charged at a pay-as-you-go pricing rate of $0.75/1,000 calls beyond their monthly limit. Alternatively, paid teams can opt for purchasing  monitoring blocks at a price of $20 for a pack of 50,000 calls, per month.

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 3]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
 
-## Running collections in a monitor (vs. the Postman app collection runner)
+Usage is calculated by number of requests made. For example, if your collection has 10 requests, but utilizes [postman.setNextRequest()](/docs/postman/collection-runs/building-workflows/) to skip several of them or run others multiple times, you'll be billed according to the total number of requests made to the server.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/postman/monitors/intro-monitors.md]: Monitoring usage is calculated by number of requests made to the server. For example, if your collection has 10 requests, and all the 10 requests are run by the monitor, you will be billed for 10 monitoring calls. However, if your collection uses [postman.setNextRequest()](/docs/postman/collection-runs/building-workflows/) to skip 4 of these requests, then you will only be billed for the remaining 6 monitoring calls (10-4 = 6). Alternatively, if your collection use [postman.setNextRequest()](/docs/postman/collection-runs/building-workflows/) to run 5 of these requests twice , you'll be billed for 15 requests (5 + 5*2 = 15).

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 4]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
+## Running collections in a monitor
+
+There are a few differences between running collections in a Postman monitor and running them via our in-app collection runner, so take note of the following.
 
 ### Variables
 
-* Can't import existing global variables, but you can create new ones during a monitor run.
-* Global and environment variables are not persisted. If you require persisting environment variables, we recommend adding a call to update the environment variable using the [Postman API](/docs/postman/postman-api/intro-api/). The following is an [example of how to update the environment variable](https://documenter.getpostman.com/view/218543/lunch-picker/6fWy4Ao#fe7e2416-4af9-fffc-02af-b8fc2c58a181) in this manner.
+* You cannot import existing global variables to a monitor, but you can create new ones during a run.
+* Global and environment variables can be updated and subsequently used during a monitoring run, however they will immediately revert to their original values following, unlike they would in the collection runner with [persist variables](/docs/postman/collection-runs/starting-a-collection-run/#persist-variables) enabled.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/postman/monitors/intro-monitors.md]: Global and environment variables can be updated and subsequently used during a monitoring run, however they will immediately revert to their original values, unlike they would in the collection runner with [persist variables](/docs/postman/collection-runs/starting-a-collection-run/#persist-variables) enabled.

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 5]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
 
-### Multi-region Monitoring
+* Monitors only run 1 iteration by default, but you can use setNextRequest() to run multiple iterations
 
-* Monitors allow you to run collections in specified geographic regions
-* Can only specify multi-region monitoring from the [monitoring page](https://monitor.getpostman.com)
+### Multi-region monitoring
 
-### Data Files
+* When setting up or editing a monitor, you can select geographic regions you'd like your monitor to run from, or opt to auto-select a region.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/postman/monitors/intro-monitors.md]: When setting up or editing a monitor, you can select multiple geographic regions you'd like your monitor to run from, or opt to auto-select a region.

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 6]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
-    name: "Intro to collection runs"
-    url: "/docs/postman/collection-runs/intro-to-collection-runs/"
+    name: "From manual to automated testing: The roadblocks and the journey"
+    url: "https://medium.com/better-practices/from-manual-to-automated-testing-the-roadblocks-and-the-journey-6333dfacc5ae"
 
 warning: false
 
 ---
 
+Postman Monitoring helps you to stay up to date on the health and performance of your APIs. Within a matter of seconds, you can set up our monitoring service and integrate it into your API development pipeline.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: could we lose first person "our" here?

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 7]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
+
+## Contents
+
+* [What is monitoring?](/docs/postman/monitors/intro-monitors/#what-is-monitoring)
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: same as last one with the relative links, if we just go from the #

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 8]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
+    * [Accessible APIs](/docs/postman/monitors/intro-monitors/#accessible-apis)
+
+* [Monitor dashboard](/docs/postman/monitors/intro-monitors/#monitor-dashboard)
+
+* [Next steps](/docs/postman/monitors/intro-monitors/#next-steps)
+
 ## What is monitoring?
 
-Postman monitoring lets you run a [collection](/docs/postman/collections/creating-collections/) periodically to check for its performance and response. You can set up a monitor to run as frequently as 5 minutes to check if all the requests in your collection are up and healthy.
+Monitoring is a way to stay up to date on the health and performance of your APIs. Postman's built-in monitoring service helps you consolidate yet another step in your API development lifecycle.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: I'd change "yet another" as it's a wee bit markety, even just "an additional step" or something..

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 9]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
+
+Postman monitors are based on [collections](/docs/postman/collections/intro-to-collections/). They can be scheduled as frequently as every five minutes and will run through each request in your collection, similar to the [collection runner](/docs/postman/collection-runs/intro-to-collection-runs/). You can also attach a corresponding [environment](/docs/postman/variables-and-environments/variables/#environments-in-postman) with variables you'd like to utilize during the collection run.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: this is a total nitpick but I'd change the "They" at start of second sentence to "Monitors" just to be extra explicit in case folk think it refers to collections or something

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 10]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
+
+The value of monitors lies in your [test scripts](/docs/postman/scripts/test-scripts/). When running your collection, a monitor will use your tests to validate the responses it's receiving. When one of these tests fail, you can automatically receive an email [notification](/docs/postman/notifications/#monitor-notifications) or configure our available [integrations](/docs/postman-pro/integrations/intro-integrations/) to be alerted in tools like Slack, PagerDuty, or HipChat.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: if we can change "our" again here

Comment (SueSmith) [reply src/pages/docs/postman/monitors/intro-monitors.md]: I'd maybe adjust "to be alerted" to "to receive alerts"

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 11]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
+
+## Pricing
 
-When you set up a monitor, Postman servers will hit the endpoints in your collection according to the specified frequency. You can also select a corresponding [environment](/docs/postman/variables-and-environments/variables/#environments-in-postman) to use and store variables. If you have written [tests](/docs/postman/scripts/test-scripts/) for your requests, the monitor would run these tests to validate the response and notify you when a test fails. You can configure how to receive the alerts from a wide number of [integrations](/docs/postman-pro/integrations/intro-integrations/) available.
+All plans, paid and free, are able to use Postman's monitoring service. To find an account type's usage limits, refer to your [usage overview](https://go.pstmn.io/postman-account-limits) or our [pricing](https://www.getpostman.com/pricing).
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: Maybe "You can use the Postman monitoring service with any paid or free plan type."

File: src/pages/docs/postman/monitors/intro-monitors.md (hunk 2 above)
Comment (SueSmith) [reply src/pages/docs/postman/monitors/intro-monitors.md]: @claireriley When we refer to account stuff I'd try to word it to address the user if that makes sense, like "With a free Postman account you can..." instead of the account doing the action.

File: src/pages/docs/postman/monitors/intro-monitors.md (hunk 4 above)
Comment (SueSmith) [reply src/pages/docs/postman/monitors/intro-monitors.md]: @claireriley I'd lose "they would" here

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 12]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
 
 ### Variables
 
-* Can't import existing global variables, but you can create new ones during a monitor run.
-* Global and environment variables are not persisted. If you require persisting environment variables, we recommend adding a call to update the environment variable using the [Postman API](/docs/postman/postman-api/intro-api/). The following is an [example of how to update the environment variable](https://documenter.getpostman.com/view/218543/lunch-picker/6fWy4Ao#fe7e2416-4af9-fffc-02af-b8fc2c58a181) in this manner.
+* You cannot import existing global variables to a monitor, but you can create new ones during a run.
+* Global and environment variables can be updated and subsequently used during a monitoring run, however they will immediately revert to their original values following, unlike they would in the collection runner with [persist variables](/docs/postman/collection-runs/starting-a-collection-run/#persist-variables) enabled.
+    * If you require persistent variables, we recommend adding a call to update your environment using the [Postman API](/docs/postman/postman-api/intro-api/).
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: if we can lose the first person "we" here, I know it can be awkward when it's a recommendation.. you could even just make it "add a call..."

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 13]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
 
-[![persist env in monitor](https://assets.postman.com/postman-docs/monitorPersistEnv.png)](https://assets.postman.com/postman-docs/monitorPersistEnv.png)
+### Console output
 
-### Console Output
+* Request and response bodies are not logged to the console for your security and privacy.
+* Headers, as they may include items like cookies and authorization keys, are also not logged.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: could we rearrange the clauses here to simplify, e.g. "Postman will also not log headers, as they may include..." maybe reword both to avoid passive..

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 14]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
 
-### Multiple Iterations
+* You cannot attach data files, unlike in the collection runner, but you can access them via APIs, including Google Docs, Google Sheets, and Dropbox.
 
-* Monitors only run 1 iteration by default
-* But you can use setNextRequest() to do multiple iterations
+### Multiple iterations
 
-### Multi-region Monitoring
+* Monitors only run 1 iteration by default, but you can use setNextRequest() to run multiple iterations
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: let's spell out words up to ten, so "one" - think we're missing a period at the end here too

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 15]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
-* Monitors allow you to run collections in specified geographic regions
-* Can only specify multi-region monitoring from the [monitoring page](https://monitor.getpostman.com)
+### Multi-region monitoring
 
-### Data Files
+* When setting up or editing a monitor, you can select geographic regions you'd like your monitor to run from, or opt to auto-select a region.
 
-* Can't attach data files like you can in the runner
-* But you can access data files from APIs, such as Google Docs, Google Sheets, Dropbox, etc.
+> If you’re interested in a region that’s not listed in the Postman interface, contact our [support team](https://support.getpostman.com/).
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: "the" instead of "our" should be fine here

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 16]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
+Each workspace has its own monitor dashboard, which you can navigate to by visiting your [web dashboard](https://go.postman.co/), selecting a workspace > **Monitors**.
 
-For Postman Enterprise users, there is an additional option available to establish dedicated IP addresses for whitelisting and source logging to address security requirements for their team. For example, by setting up a static IP address for API monitoring, you can whitelist the dedicated IPs with your incoming servers to ensure they will accept requests from those IP addresses. Postman Enterprise users interested in [setting up static IPs](/docs/postman-enterprise/using-static-IPs-to-monitor/) for their API monitoring should contact [help@getpostman.com](mailto:help@getpostman.com).
+This dashboard provides a high-level overview of the monitors you have available in your workspace: their status, success rate, and average response time.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: I'd maybe just go with "...your workspace, including status..." instead of the colon

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 17]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
 
-## Pricing for monitoring
+[![monitoring dashboard](https://assets.postman.com/postman-docs/monitor-dashboard1.jpg)](https://assets.postman.com/postman-docs/monitor-dashboard.jpg)
 
-Monitoring is priced per request made, with some free requests included every month. Learn more about [monitoring pricing](/docs/postman/monitors/pricing-monitors/).
+Hovering over a monitor in the list allows you to run it outside of its predetermined schedule. It also allows you via the **...** to pause, resume, edit, and delete monitors.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: Maybe rearrange the second sentence to "To pause, resume, edit, and delete monitors, use **...**." (that thing where you frontload the info so people know if the sentence is relevant to them from the start..)

The "..." buttons are hard to refer to, if it has a name e.g. text that pops up on hover I sometimes use that as well.

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 18]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
 
-## Free monitoring calls with your Postman account
+## Next steps
 
-Your Postman account gives you a limited number of free monitoring calls per month. You can check your usage limits through the [Postman API](https://docs.api.getpostman.com) or the [account usage page](https://go.pstmn.io/postman-account-limits).
+Learn how to [set up a monitor](/docs/postman/monitors/setting-up-monitor/) and check out our page on [monitoring APIs and websites](/docs/postman/monitors/monitoring-apis-websites/) to get started.
```
Comment (SueSmith) [inline src/pages/docs/postman/monitors/intro-monitors.md]: one last wee "our" to get rid of 😬

**Comment by SueSmith** [comment]
also took me a while to catch on but realised if I do comments via "start review" instead of individual ones they don't generate multiple notifications so hopefully won't drive people watching up the wall with emails (except this one of course) 🙃

File: src/pages/docs/postman/monitors/intro-monitors.md (hunk 3 above)
Comment (claireriley) [reply src/pages/docs/postman/monitors/intro-monitors.md]: Moving this explanation to this page: https://learning.getpostman.com/docs/postman/monitors/pricing-monitors/ as I don't want to get too redundant here

File: src/pages/docs/postman/monitors/intro-monitors.md (hunk 6 above)
Comment (claireriley) [reply src/pages/docs/postman/monitors/intro-monitors.md]: 🤦‍♀ too used to speaking as a company representative from support/sales/success/marketing

File: src/pages/docs/postman/monitors/intro-monitors.md (hunk 17 above)
Comment (claireriley) [reply src/pages/docs/postman/monitors/intro-monitors.md]: Unfortunately no text on hover. Let me know what you think of the update (didn't want to end the sentence with **...** as **...**. looked weird)

**Comment by claireriley** [comment]
@SueSmith I think I covered everything—let me know!

File: src/pages/docs/postman/monitors/intro-monitors.md (hunk 6 above)
Comment (SueSmith) [reply src/pages/docs/postman/monitors/intro-monitors.md]: lol it's definitely not a bad perspective to have

File: src/pages/docs/postman/monitors/intro-monitors.md (hunk 17 above)
Comment (SueSmith) [reply src/pages/docs/postman/monitors/intro-monitors.md]: lgtm, aye it's a tricky one, we might want to have a think about how to handle descriptions of ... buttons in general and stick it in the style guide

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 19]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
-* But you can access data files from APIs, such as Google Docs, Google Sheets, Dropbox, etc.
+> If you’re interested in a region that’s not listed in the Postman interface, contact the [Postman support team](https://support.getpostman.com/).
 
 ### Accessible APIs
 
-* As with the Postman app, monitoring requires all URLs to be publicly-available on the Internet. In the future, you will be able to monitor private APIs as well.
-* Monitors can't directly access your `localhost` and might encounter a firewall because monitoring runs in the Postman cloud.
-
-## Monitoring resources in multiple regions
+* Monitors require all URLs to be publicly-available on the internet, and cannot directly access your `localhost`. Monitors run in the Postman cloud, so you may encounter a firewall. Static IPs are available for [certain plan types](https://www.getpostman.com/pricing).
```
Comment (ArjunSingh-PM) [inline src/pages/docs/postman/monitors/intro-monitors.md]: Since a monitor runs in the Postman cloud, it require all URLs to be publicly-available on the internet. A monitor cannot directly access your `localhost` or run requests behind a firewall. However, to overcome this issue,  [enterprise users](https://www.getpostman.com/pricing) can configure Static IPs for running their monitors.

File: src/pages/docs/postman/monitors/intro-monitors.md [hunk 20]
```diff
@@ -23,88 +23,130 @@ contextual_links:
...
+This dashboard provides a high-level overview of the monitors you have available in your workspace, including status, success rate, and average response time.
 
-## Pricing for monitoring
+[![monitoring dashboard](https://assets.postman.com/postman-docs/monitor-dashboard1.jpg)](https://assets.postman.com/postman-docs/monitor-dashboard.jpg)
 
-Monitoring is priced per request made, with some free requests included every month. Learn more about [monitoring pricing](/docs/postman/monitors/pricing-monitors/).
+Hovering over a monitor in the list allows you to run it outside of its predetermined schedule. To pause, resume, edit, and delete monitors, select the **...** icon.
```
Comment (ArjunSingh-PM) [inline src/pages/docs/postman/monitors/intro-monitors.md]: Hovering over a monitor in the list allows you to run it outside of its predetermined schedule using the **[>]** icon. To pause, resume, edit, and delete monitors, select the **[...]** icon.
